### PR TITLE
Update copyright year

### PR DESCRIPTION
**Pull request by sandakersmann**
Update copyright year to 2023

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2019 Truffle Blockchain Group, Inc.
+Copyright (c) 2015-2023 Truffle Blockchain Group, Inc.
```
Comment (cliffoo) [inline LICENSE:1]:
```suggestion
Copyright (c) 2015-2023 ConsenSys Software Inc.
```

**Review by cliffoo — CHANGES_REQUESTED** [review]
Thanks @sandakersmann

File: LICENSE (hunk 1 above)
Comment (sandakersmann) [reply LICENSE:1]: I think you can make a separate PR for this, since I have no clue if this change is legal.

Comment (gnidan) [reply LICENSE:1]: TBG transferred all IP to ConsenSys Software Inc as part of the acquisition.

Comment (cliffoo) [reply LICENSE:1]: > I have no clue if this change is legal.

It is. Oh well I figured you might want to address two lapses in the text in one PR.

**Review by cliffoo — APPROVED** [review]

**Review by eggplantzzz — APPROVED** [review]

**Pull request merged**